actionpack/test/dispatch/cookies_test.rb

```diff
@@ -278,6 +278,21 @@ def noop
     def encrypted_cookie
       cookies.encrypted["foo"]
     end
+
+    def set_expiration_using_expires_in
+      cookies["user_name"] = { value: "assain", expires_in: 1.hour }
+      head :ok
+    end
+
+    def set_expiration_using_expires_at
+      cookies["user_name"] = { value: "assain", expires_at: Time.now.advance(years: 1)}
+      head :ok
+    end
+
+    def set_signed_cookie_expiration_using_expires_in
+      cookies.signed["user_name"] = { value: "assain", expires_in: 1.hour }
+      head :ok
```

Comment:
Don't controllers respond with `head :no_content` by default? So that these `head` lines can be omitted?